src/components/Nav/Nav.js

```diff
@@ -1,9 +1,97 @@
+import React from 'react';
 import styled from 'styled-components';
+import variables from '../../styles/variables';
+import MenuBar from '../../assets/Nav/MenuBar.png';
+import logo from '../../assets/Nav/logo.png';
 
 export default function Nav() {
-  return <Title>Nav입니다.</Title>;
+  return (
+    <Container>
+      <Title>
+        <LeftMenu />
+        <Logo />
+        <Menu>
+          <li>
+            <a href=" ">채용</a>
+          </li>
+          <li>
+            <a href=" ">이벤트</a>
+          </li>
+          <li>
+            <a href=" ">직군별 연봉</a>
+          </li>
+          <li>
+            <a href=" ">커뮤니티</a>
+          </li>
+          <li>
+            <a href=" ">프리랜서</a>
+          </li>
+          <li>
+            <a href=" ">AI 합격예측</a>
+          </li>
+        </Menu>
+        <Aside>
+          <Menu>
+            <li>
+              <a href=" ">회원가입/로그인</a>
+            </li>
+          </Menu>
+        </Aside>
+      </Title>
+    </Container>
+  );
 }
 
-const Title = styled.nav`
+const Container = styled.div`
+  width: 100%;
+  height: 50px;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+  top: 0;
   box-shadow: ${props => props.theme.selectedBoxShadow};
+  position: fixed;
+  background-color: #fff;
+  list-style-type: none;
+`;
+
+const Title = styled.nav`
+  display: flex;
+  margin: 0 auto;
+  flex-wrap: wrap;
+  height: 50px;
+  width: 90%;
+  font-size: 14px;
+  ${variables.flex(' ', 'space-between')};
 `;
+
+const LeftMenu = styled.img.attrs({
+  src: `${MenuBar}`,
+})`
+  width: 20px;
+  height: 17px;
+  ${variables.flex()};
+`;
+
+const Logo = styled.img.attrs({
+  src: `${logo}`,
+})`
+  height: 17px;
+`;
+
+const Menu = styled.ul`
+  text-align: center;
+  margin: 0 auto;
+
+  li {
+    padding: 10px;
+    float: left;
+
+    a {
+      text-decoration: none;
+      color: ${props => props.theme.selectedBlack};

```

Comment:
```suggestion
      color: ${({theme})=> theme.selectedBlack};
```
매개변수자리에서 구조분해 할당 할 수 있습니다

Reply:
감사합니다!